lib/internal/validators.js

```diff
@@ -324,6 +324,21 @@ function validateBooleanArray(value, name) {
   }
 }
 
+/**
+ * @callback validateAbortSignalArray
+ * @param {*} value
+ * @param {string} name
+ * @returns {asserts value is AbortSignal[]}
+ */
+
+/** @type {validateAbortSignalArray} */
+function validateAbortSignalArray(value, name) {
+  validateArray(value, name);
+  for (let i = 0; i < value.length; i++) {
+    validateAbortSignal(value[i], `${name}[${i}]`, false);
```

Comment:
```suggestion
	if (value[i] == null) {
		throw new ERR_INVALID_ARG_TYPE(`${name}[${i}]`, 'AbortSignal', signal);
	}
    validateAbortSignal(value[i], `${name}[${i}]`);    
```
I thinks this is better then the addition to `validateAbortSignal`